package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-icons",
-  "version": "0.0.12",
+  "version": "0.0.13",
```

Comment:
Version changes should be done using separate commit and PR.

Reply:
Ah, forgot to change that back after testing with verdaccio.